### ISSUE TITLE
No usage detected for some functions

### ISSUE THREAD

**Issue by OscarAkaElvis** [BashSupport-Pro/bashsupport-pro#82]
Hi, after some bashsupport-pro update, some usage functions are not detected. I test it using [airgeddon](https://github.com/v1s1t0r1sh3r3/airgeddon) which is a big script.

A couple of examples

![image](https://user-images.githubusercontent.com/5803348/127854554-16024040-ca02-4638-87a0-40f32e48c7d0.png)

![image](https://user-images.githubusercontent.com/5803348/127854560-2a32efac-e4ef-4556-ace1-a29598681938.png)

This used to work, I'm 99% sure that it stopped working after major update. Cheers.

**Comment by jansorg** [comment]
Thanks! I can confirm that this isn't working. Also, code analysis is very slow with the airgeddon script. I'll investigate and should have a fix for the next update.

**Comment by jansorg** [comment]
There were issues with the parsing of two conditional commands. I've fixed this for the next update and also slightly reduced memory usage for faster highlighting and indexing.

**Comment by OscarAkaElvis** [comment]
Awesome news, thanks for your hard work.

Should we close this? or wait until new version is released?

**Comment by jansorg** [comment]
I'll close this when the new update is available.

**Comment by jansorg** [comment]
Version 2.0.4 with the fixes is now available.

**Comment by OscarAkaElvis** [comment]
Weird, I tried the update on my Pycharm and it failed:

![image](https://user-images.githubusercontent.com/5803348/128542272-f2af9688-cb6e-43aa-b09a-a6f34b62cdaa.png)

Then after restarting IDE, the plugin was not installed. I installed it again from the embedded market and now it worked. And yes, what is on this thread is fixed now. My version of the plugin now is 2.0.4.212. 

Awesome work, thanks!
